sklearn/neighbors/base.py

```diff
@@ -417,6 +407,26 @@ class from an array representing our data set and ask who's
                 return dist, neigh_ind
             return neigh_ind
 
+    def _reduce_func(self, dist, n_samples, **kwds):
+        sample_range = np.arange(n_samples)[:, None]
```

Comment:
I use that myself too. Did this here as `sample_range` is defined in `kneighbors` in the same way.